test/__snapshots__/react.test.js.snap

```diff
@@ -520,15 +537,15 @@ exports[`styled random object expression 1`] = `
 `;
 
 exports[`styled themes 1`] = `
-.glamor-1 {
+.glamor-3 {
   background-color: #ffd43b;
   color: #8c81d8;
   height: 64px;
   font-size: 32px;
 }
 
 <span
-  className="glamor-0 glamor-1"
+  className="glamor-0 glamor-1 glamor-2 glamor-3"
```

Comment:
Yes, that's correct because they're wrapped in `withTheme` so more components are rendered and the class is added for using components as selectors.

Now that I think about it, we might want to always add the class for using components as selectors because in this example  `OneComponent` will have `color: yellow;` when rendered inside of `SomeComponent` but `AnotherComponent` will not.
```jsx
const OneComponent = styled.div`
  display: flex;
`
const AnotherComponent = styled(OneComponent)`
  background-color: hotpink;
`
const SomeComponent = styled.div`
  ${OneComponent} {
    color: yellow;
  }
`